mllib/src/main/scala/org/apache/spark/ml/clustering/GaussianMixture.scala

```diff
@@ -105,6 +108,15 @@ class GaussianMixtureModel private[ml] (
   def gaussians: Array[MultivariateGaussian] = parentModel.gaussians
 
   @Since("2.0.0")
```

Comment:
@jkbradley As I discussed with Yanbo, I only keep one method, gaussians, instead of two separate: gaussians and gaussiansDF. Do you mean renaming gaussians to gaussiansDF?
Thanks!